exorcist/taskdb.py

```diff
@@ -1,11 +1,16 @@
 import abc
 import sqlalchemy as sqla
+import networkx as nx
+from .models import TaskStatus
 
 # remaining imports are for typing
 from typing import Optional, Iterable
-from .models import TaskStatus
 from os import PathLike
-import networkx as nx
+
+def sqlite_fk_pragma(dbapi_conn, conn_record):
+    cursor = dbapi_conn.cursor()
+    cursor.execute("PRAGMA foreign_keys=ON")
+    cursor.close()
```

Comment:
document what this does and why it's necessary, probably make it hidden behind an underscore